CONDUCT.rst

```diff
@@ -0,0 +1,45 @@
+=================
+ Code of Conduct
+=================
+
+Welcome
+=======
+Contributors and users of armasec come from folks from around the globe
+with a diverse set of skills, personalities, and experiences. It is through
+these differences that our community experiences great successes and continued
+growth. When you're working with other contributors or users, we expect you
+to follow these guidelines which help steer our interactions and strive to keep
+the development of armasec positive, successful, and growing.
+
+Be Open
+=======
+Contributors to armasec are expected to be open. They should be open to
+collaboration, whether it's on issues, pull-requests, or otherwise. We're
+receptive to constructive comment and criticism, as the experiences and skill
+sets of other members contribute to the whole of our efforts. We're accepting of
+all who wish to take part in our activities, fostering an environment where
+anyone can participate and everyone can make a difference.
+
+Be Considerate
+==============
+Contributors to armasec are expected to be considerate of their peers.
+We're thoughtful when addressing the efforts of others, keeping in mind that
+often times the labor was completed simply for the good of the community. We're
+attentive in our communications, whether in person or online, and we're tactful
+when approaching differing views.
+
+Be Respectful
+=============
+Contributors to armasec are expected to be respectful. We're respectful
+of others, their positions, their skills, their commitments, and their efforts.
+We're respectful of competing packages and others in the Flask and Python
+community at large.  We're respectful of the processes set forth in the Python
+community, and we work within them. When we disagree, we are courteous in
+raising our issues.
+
+Conclusion
+==========
+Overall, we're good to each other. Contribution is done on a voluntary basis,
+and folks are expected to be good and decent in their conduct. Any disputes
+will be resolved by the maintainers and those that do not wish to follow the
+guidelines will be asked to stop contributing.
```

Comment:
9 occurrences of `we're`. What else can we use instead of `we're` or `we are`?

Reply:
Overall, really good stuff!

Reply:
Changed we're to we are in next commit.

Also, wish I could claim credit, but this is a slight adaptation of the Python code of conduct.